packages/core/saved-objects/core-saved-objects-api-server/src/apis/create.ts

```diff
@@ -61,4 +61,6 @@ export interface SavedObjectsCreateOptions extends SavedObjectsBaseOptions {
    * * For global object types (registered with `namespaceType: 'agnostic'`): this option cannot be used.
    */
   initialNamespaces?: string[];
+  /** Optional flag to indicate a document is managed. */
```

Comment:
```suggestion
/**
   * Flag indicating if a saved object is managed by Kibana (default=false)
   *
   * This can be leveraged by applications to e.g. prevent edits to a managed
   * saved object. Instead, users can be guided to create a copy first and
   * make their edits to the copy.
   */
```